docs/source/sim_overview.rst

```diff
@@ -1,43 +1,41 @@
 Overview
 ========
 
+.. role:: strike
+
+Why run a simulation?
+---------------------
+
+Doing simulations can be a great way to assess how well ASReview performs for your particular purposes. The user can run simulations on previously labelled datasets to see - for example - how much time the user could have saved using ASReview, or how previous inclusion criteria could be improved in the future. The ASReview simulation mode works by sequentially going through the dataset like an ASReview user would, and uses the labels to learn from these inclusions and exclusions. In this way, the entire screening process is replicated.
 
 Doing the simulation
 --------------------
 
-Doing simulations can be a great way to assess how well *ASReview* performs for your particular
-needs.
-There are a few ways to perform simulations with the ASReview. For how to prepare your data,
-see the `dataset <datasets.html>`__ page.
-
-The first option is to use the simulation mode that is provided with the _ASReview_ package. It can
+You can use the simulation mode that is provided with the ASReview package. It can
 be accessed directly from the command line, for example:
 
 .. code-block:: bash
 
-	asreview simulate YOUR_DATASET.csv --state_file myreview.h5 --n_instances 1 -m nb
+	asreview simulate MY_DATASET.csv --state_file myreview.h5
+
+This performs a simulation of a default active learning model, where ``MY_DATASET.csv`` is the path to the dataset you wish to simulate on and where ``myreview.h5`` is the file where the results will be stored:
 
-This performs a simulation retraining the model after reviewing each record with the Naive Bayes
-model, and storing the results in the file ``myreview.h5``. More detail can be found in the section
-on the `cli simulation <cli.html>`__ page.
+.. code-block::
 
-A second option is to use the functionality of the 
-`asreview-simulation <https://github.com/asreview/automated-systematic-review-simulations>`_
-package. It makes it easier to run multiple simulations at the same time in case that we have
-more cores/processors available. Notice that it requires a library that implements the MPI standard,
-such as OpenMPI.
+	├── my-simulation
```

Comment:
I think this level will be confusing to users not so familiar with the command line. If you want a nested directory structure, then I suggest we put it in the command as well.

Reply:
There is no need to put any directory structure in the command line, right? 
I made up a folder called 'my-simulation' to make clear that the data and the state file are in the same directory. Do you want to get rid of this folder? Do you suggest leaving that whole thing out?

Reply:
Yes, I would leave that folder out, since users might try to run the command from the wrong folder. Another solution might be to put in an explicit folder change in the documentation.